Source/Libs/WebkitGtkSharp/WebkitGtkSharp-api.xml

```diff
@@ -0,0 +1,6011 @@
+            <property name="CanAttach" cname="can-attach" type="gboolean" readable="true" writeable="false" construct="false" construct-only="false" version="2.8"/>
+            <property name="InspectedUri" cname="inspected-uri" type="gchar*" readable="true" writeable="false" construct="false" construct-only="false"/>
+            <field cname="parent" access="public" writeable="false" readable="true" is_callback="false" name="Parent" type="GObject*"/>
+            <field cname="priv" access="public" writeable="false" readable="true" is_callback="false" name="Priv" type="WebKitWebInspectorPrivate*">
+                <warning>missing glib:type-name</warning>
+            </field>
+            <signal name="AttachEvent" cname="attach" when="last">
+                <return-type type="gboolean"/>
+                <parameters/>
+                <warning>Signal renamed because of existing method with same name</warning>
+            </signal>
+            <signal name="BringToFront" cname="bring-to-front" when="last">
+                <return-type type="gboolean"/>
+                <parameters/>
+            </signal>
+            <signal name="Closed" cname="closed" when="last">
+                <return-type type="void"/>
+                <parameters/>
+            </signal>
+            <signal name="DetachEvent" cname="detach" when="last">
+                <return-type type="gboolean"/>
+                <parameters/>
+                <warning>Signal renamed because of existing method with same name</warning>
+            </signal>
+            <signal name="OpenWindow" cname="open-window" when="last">
+                <return-type type="gboolean"/>
+                <parameters/>
+            </signal>
+        </object>
+        <object name="WebResource" cname="WebKitWebResource" opaque="false" hidden="false" parent="GObject">
+            <class_struct cname="WebKitWebResourceClass">
+                <field cname="parent_class" access="public" writeable="false" readable="true" is_callback="false" name="ParentClass" type="GObjectClass">
+                    <warning>missing glib:type-name</warning>
+                </field>
+                <method vm="_webkit_reserved0"/>
+                <method vm="_webkit_reserved1"/>
+                <method vm="_webkit_reserved2"/>
+                <method vm="_webkit_reserved3"/>
+            </class_struct>
+            <virtual_method name="_webkitWebkitReserved0" cname="_webkit_reserved0" shared="true" hidden="true">
+                <return-type type="void"/>
+            </virtual_method>
+            <virtual_method name="_webkitWebkitReserved1" cname="_webkit_reserved1" shared="true" hidden="true">
+                <return-type type="void"/>
+            </virtual_method>
+            <virtual_method name="_webkitWebkitReserved2" cname="_webkit_reserved2" shared="true" hidden="true">
+                <return-type type="void"/>
+            </virtual_method>
+            <virtual_method name="_w
```

Comment:
definition is here, why it's not creating the class??

Reply:
GapiCodegen can't find interfaces, don't know why.

Remove/comment in xml (2 instances):
```xml
<implements>
    <interface cname="Atk.ImplementorIface"/>
    <interface cname="GtkBuildable"/>
</implements>
```

Use metadata:
```xml
<metadata>
    <attr path="/api/namespace/object[@cname='WebKitAuthenticationRequest']/method[@name='CanSaveCredentials']" name="name">GetCanSaveCredentials</attr>
</metadata>
```

There are new errors...

Reply:
Need to fix only `webkit_form_submission_request_list_text_fields` somehow. Then compile succeed.

Reply:
@zii-dmg thanks, compiles & sample works (with some flaws)
but this will get extra pr's